CONTRIBUTING.md

```diff
@@ -1,19 +1,47 @@
-# Don't use the issue tracker to ask questions
```

Comment:
Maybe start with a table of contents like 'Issues / Basic setup / Acceptance tests' so it's obvious that it's not just generic bla-bla / users can skip to the section they need